lesson-02-assignment/lesson-02-assignment.rb

```diff
@@ -1,16 +1,19 @@
 #1
 
-Course.create(coding_class_id: "1", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "2", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "3", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "4", trimester_id: "8", max_enrollment: "25")
-Course.create(coding_class_id: "5", trimester_id: "8", max_enrollment: "25")
+(1..5).each do |id|

```

Comment:
Instead of looping through numbers 1 through 5, practice getting all the coding classes and looping through them (since the `id` for a coding classes could be anything).  So instead of using `(1..5)` you would use something like this:
```
CodingClass.all.each do |coding_class|
  ...
end
```